python/lnotify.py

```diff
@@ -101,9 +101,9 @@ def handle_msg(data, pbuffer, date, tags, displayed, highlight, prefix, message)
         cmd_name=("ps -ho comm -p %s"%(window_pid)).split()
         window_name = subprocess.check_output(cmd_name)
         ignore_windows_list = ["tilda", "gnome-terminal", "xterm"]
-    if ignore_windows_list in window_name:
-        x_focus = True
-        return weechat.WEECHAT_RC_OK
+        if ignore_windows_list in window_name:
```

Comment:
shouldn't it be the other way around, `window_name in ignore_window_list` anyway

Reply:
Uh, probably, yes. I just fixed the indentation because it causes an exception.